app/views/users/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is a list of Users</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag 'users_css' %>
+  </head>
+
+  <body>
+    <header>
+      <h1>This is the list of Users</h1>
+    </header>
```

Comment:
- [ ] I think it would be better to remove the header as it is not given in the wireframe. You can style it the way you want but please make sure that it matches the given wireframe ⛏🤓


Current | Given 
---| ---|
![image](https://github.com/oovillagran/blog_app/assets/106119105/43322f01-8668-4e08-93b0-36ca86a2437f) | ![image](https://github.com/oovillagran/blog_app/assets/106119105/bdda02dd-f323-4a6f-8604-5ae2dcccc5d4)

Reply:
Done!